app/components/date/custom-date-picker.tsx

```diff
@@ -104,3 +104,113 @@ export const CustomDatePicker = ({
     </Box>
   )
 }
+
+export const CustomDateTimePicker = ({
```

Comment:
For smartphone looks broken a little bit  👀 
Could you set aside or expand the time field to match the calendar? 
<img width="408" alt="Screenshot 2024-02-26 at 12 26 41 PM" src="https://github.com/tabi-memo/frontend/assets/66394413/35cbbc75-9946-4b04-b5b8-4c15789fcf73">

Reply:
Fixed by https://github.com/tabi-memo/frontend/pull/34/commits/df7c7e6c6829b7aa96e859a7137cd5bd01e444d6

| Mobile | PC |
|--------|----|
| ![Screenshot 2024-03-01 at 15 41 47](https://github.com/tabi-memo/frontend/assets/7115171/23546a0f-f40e-40c0-b0a5-57da91c73610) | ![Screenshot 2024-03-01 at 15 42 11](https://github.com/tabi-memo/frontend/assets/7115171/f302078a-f1d1-4e73-9eed-2b2cb0aa7a6d) |